packages/framework-integration-tests/package.json

```diff
@@ -45,7 +45,7 @@
     "metadata-booster": "0.3.1",
     "mocha": "8.4.0",
     "mocha-skip-if": "0.0.3",
-    "serverless": "3.8.0",
+    "serverless": "1.83.3",
     "serverless-artillery": "0.5.2",
```

Comment:
So bad we're still relying on this, as this package has been abandoned. Just as a reminder for ourselves, there's a pending issue related to finding an alternative or building this capability within Booster https://github.com/boostercloud/booster/issues/934